src/components/backend-ai-storage-list.ts

```diff
@@ -1244,6 +1244,7 @@ export default class BackendAiStorageList extends BackendAIPage {
           <mwc-icon-button
             class="fg green controls-running"
             icon="info"
+            title="info"
```

Comment:
How about using i18n strings like others?
Here's a simple way to use the i18n string.
1. Modify [ko.json or en.json](https://github.com/lablup/backend.ai-webui/tree/main/resources/i18n) files.
2. Run `make i18n`
3. Translate other words if you can.

More details: https://github.com/lablup/backend.ai-webui#extracting-i18n-resources

Reply:
Ok I will try it!

Reply:
I resolve it. Can you review again?